server/mediaserver.go

```diff
@@ -436,7 +439,10 @@ func (s *LivepeerServer) startSessionListener(cxn *rtmpConnection) {
 		}
 		mut.Lock()
 		defer mut.Unlock()
-		cxn.sess = sess
+		cxn.sessManager.broadcastSessions = sess
+		for _, s := range sess {
+			cxn.sessManager.broadcastSessMap[s.OrchestratorInfo.Transcoder] = s
+		}
```

Comment:
This part has been subsumed by the `refreshSession` call in NewSessionManager so this isn't needed anymore.